src/insert.rs

```diff
@@ -170,6 +170,7 @@ impl SupabaseClient {
             )
             .header(HeadersTypes::ContentType, "application/json")
             .header(HeadersTypes::ClientInfo, &crate::client_info())
+            .header(HeadersTypes::Prefer, "return=representation")
```

Comment:
Where is the documentation for this request header ?
Is there an official doc explaining the possible values ?

Reply:
I got this of stack overflow initially, but after some quick searching, it is documented in the PostgREST (used by supabase for db API) docs here: https://docs.postgrest.org/en/v12/references/api/preferences.html#prefer-return